### ISSUE TITLE
Nextcloud crash during synchronizing

### ISSUE THREAD

**Issue by gitaarik** [nextcloud/android#6416]
************ CAUSE OF ERROR ************

java.lang.RuntimeException: Error receiving broadcast Intent { act=com.owncloud.android.files.services.FileUploaderUPLOAD_FINISH flg=0x10 pkg=com.nextcloud.client (has extras) } in com.owncloud.android.ui.activity.FileDisplayActivity$UploadFinishReceiver@5b7385
	at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$getRunnable$0(LoadedApk.java:1401)
	at android.app.-$$Lambda$LoadedApk$ReceiverDispatcher$Args$_BumDX2UKsnxLVrE6UJsJZkotuA.run(Unknown Source:2)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6746)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.owncloud.android.datamodel.OCFile.getRemotePath()' on a null object reference
	at com.owncloud.android.ui.activity.FileDisplayActivity$UploadFinishReceiver.onReceive(FileDisplayActivity.java:1476)
	at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$getRunnable$0(LoadedApk.java:1391)
	... 8 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30120090
Build flavor: generic

************ DEVICE INFORMATION ************
Brand: Fairphone
Device: FP3
Model: FP3
Id: 8901.2.A.0120.20200421
Product: FP3

************ FIRMWARE ************
SDK: 28
Release: 9
Incremental: 04211500

**Comment by tobiasKaminsky** [comment]
Do you know how you can reproduce this?

**Comment by gitaarik** [comment]
Not completely sure, but the error has been happening since I added a 
new Auto Upload folder. That folder is quite big (my WhatsApp images 
directory). The app has to sync about 5500 new photos. So it takes quite 
some time. It also gets a lot of failed to connect errors and 
synchronization errors, I get notifications where it asks that I choose 
between two versions of a file, but when I click on the notification, 
the Nextcloud app opens on a blank page and nothing comes up. Anyway the 
app responds really slow, because it is busy synchronizing all those 
files. Sometimes Android tells me that the app is keeps crashing and 
asks me if I want to force close it or wait. Then sometimes it crashes 
and I get this crash report that I just submitted.
